### ISSUE TITLE
关于CogVideoX-Fun-V1.1-2b-InP模型

### ISSUE THREAD

**Issue by thbupt** [aigc-apps/CogVideoX-Fun#118]
这个模型是基于cogvideox的i2v模型微调得到的吗，但我看其中tranformers的部分的in_channels是33，但原始模型是32。还有个问题，在i2v中，在对输入的视频进行vae编码之前，是把非首帧的值设置为了-1，但感觉设置成0更好一些？
masked_video = init_video * (mask_condition_tile < 0.5) + torch.ones_like(init_video) * (mask_condition_tile > 0.5) * -1

**Comment by endic-sam928281** [comment]
Hello, we tried to solve the issue.

This is what we did:

1. In transformer3d.py, change the in_channels parameter of CogVideoXTransformer3DModel from 16 to 33.
2. In pipeline_cogvideox.py, update the masking logic to use 0 instead of -1 for non-first frames.

You can review changes in this commit: https://github.com/endic-sam928281/aigc-apps-CogVideoX-Fun-118/commit/ef3c3cba78922cf8e7765ebcba2d85384c97ff32.


> [!CAUTION]
> Disclaimer: The concept of solution was created by AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---

Latta AI seeks to solve problems in open source projects as part of its mission to support developers around the world. Learn more about our mission at https://latta.ai/ourmission . If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.

**Comment by thbupt** [comment]
@endic-sam928281 Thanks for your reply. But if the in_channels of transformer3d is changed, the model need to be trained from scratch?
